spine-ts/spine-player/example/example-audio.html

```diff
@@ -0,0 +1,38 @@
+<!doctype html>
+<html>
+
+<head>
+	<meta charset="utf-8">
+	<script src="../dist/iife/spine-player.js"></script>
+	<link rel="stylesheet" href="../css/spine-player.css">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+
+<style>
+	body {
+		background: gray;
+		margin: 0px;
+	}
+</style>
+
+<body>
+	<div id="container" style="width:640px; height:380px"></div>
+</body>
+<script>
+	// Creates a new spine player. The debugRender option enables
+	// rendering of viewports and padding for debugging purposes.
+	new spine.SpinePlayer("container", {
```

Comment:
See comment above. Move this to `example.html` as a third player. Remove `example-audio.html`